tango/common/from_params.py

```diff
@@ -318,9 +318,11 @@ def construct_arg(
     # The parameter is optional if its default value is not the "no default" sentinel.
     optional = default != _NO_DEFAULT
 
-    if hasattr(annotation, "from_params"):
+    if inspect.isclass(annotation) and issubclass(annotation, FromParams):
```

Comment:
This wasn't necessary for the fix, but makes this function more robust in general.